apps/settings/js/icc.js

```diff
@@ -500,12 +526,19 @@
       li.appendChild(small);
     }
 
+    if (entry.icon && STKHelper) {
```

Comment:
And also, you have to add one more line here to make sure we would make STKHelper as a shim and load them into our scripts.

 https://github.com/mozilla-b2g/gaia/blob/master/apps/settings/build/settings.build.jslike#L31